core/src/elements/ons-splitter-side.js

```diff
@@ -629,6 +629,8 @@ export default class SplitterSideElement extends BaseElement {
     const unlock = this._lock.lock();
     this._state = CHANGING_STATE;
 
+    options.animation && this._updateAnimation(options.animation);
```

Comment:
I'm not a fan of this shorthand. It can easily be read as a mistake if someone doesn't totally understand how `&&` works. I'd prefer a simple `if` statement.

Reply:
Thanks, I've made the change. This shorthand is used all over the code so if we're not going to use it, I think we should either add it to the style guide in CONTRIBUTORS.md or to eslint.

Reply:
If we can add it to the eslint file, that would be great. This code all gets put through babel and minified, so saving space is not a concern :)